CMakeLists.txt

```
@@ -49,8 +50,10 @@ set(PROJECT_SOURCES
 
 add_executable(Deckbuilder ${PROJECT_SOURCES})
     
-target_link_libraries(Deckbuilder PRIVATE Qt${QT_VERSION_MAJOR}::Widgets ${SQLite3_LIBRARIES} JsonCpp::JsonCpp)
+target_link_libraries(Deckbuilder PRIVATE Qt5::Widgets ${SQLite3_LIBRARIES} JsonCpp::JsonCpp)
 
+# database builder
+include(${PROJECT_SOURCE_DIR}/db_builder/CMakeLists.txt)
```

```suggestion
add_subdirectory(${PROJECT_SOURCE_DIR}/db_builder)
```
Łap lekturę
https://stackoverflow.com/questions/48509911/cmake-add-subdirectory-vs-include

---

@gizlu  popraw to później, poki co robie juz merga